packages/eslint-plugin-pf-codemods/lib/rules/v5/emptyStateIcon-icon-required.js

```diff
@@ -0,0 +1,30 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8737
+module.exports = {
+  meta: {},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-core").filter(
+      (specifier) => "EmptyStateIcon" === specifier.imported.name
+    );
+    return imports.length === 0
+      ? {}
+      : {
+          JSXOpeningElement(node) {
+            if (imports.find((imp) => imp.local.name === node.name.name)) {
+              const iconPropIncluded = node.attributes
+                .filter((attr) => attr.name)
+                .map((attr) => attr.name.name)
+                .includes("icon");
+
+              if (!iconPropIncluded) {
```

Comment:
```suggestion
              if ( node.attributes.find( a => ['icon', 'component'].includes(a.name?.name))) {
```

I wonder if we should also check for component here like above, as the other rule will convert component into icon... 

as it stands now, if they were using component it would say "icon is required", "component is now icon" and also convert it to icon with a fixer..

what do you guys think @thatblindgeye @wise-king-sullyman ?

currently
`70|  <EmptyStateIcon component={Spinner} variant="container"/>`
will give
<img width="690" alt="image" src="https://user-images.githubusercontent.com/5322142/225714520-42e4d1c8-cf31-4b57-ad19-8dd2f74830f6.png">

Reply:
Good point, it would be confusing to say that "icon is required", when the fixer in the other rule converts the "component" into "icon".

Reply:
Also, I have a question, why do we have to check that `.name` attribute exists on the `JSXOpeningElement`? (`a.name?.name`). I can't think of any use case where the JSXAttribute does not have a name, are there some special cases?